Problem2.java

```diff
@@ -0,0 +1,31 @@
+class Solution {
+    public int findMin(int[] nums) {
+        
+        int l=0;
+        int h=nums.length;
+        
+        while(l<h)
+        {
```

Comment:
Good Job! Please add comments, time and space complexities and the third problem for this assignment.